### PR TITLE
Add workaround for CERTIFICATE_VERIFY_FAILED error

### PR DESCRIPTION
**Pull request by nsoranzo**

### CODE REVIEW

File: topics/admin/tutorials/tool-management/tutorial.md [hunk 1]
```diff
@@ -146,6 +146,20 @@ There are two ways to install tools, depending on how you specify the tools to i
 >
 {: .hands_on}
 
+> ### {% icon tip %} Certificate issues
+>
+> If your Galaxy instance is served via the HTTPS protocol (as it should!), ephemeris will use the [requests](https://requests.readthedocs.io) Python library to encrypt the communication with Galaxy. Therefore, if your Galaxy uses a self-signed SSL certificate, `shed-tools` may fail with a `CERTIFICATE_VERIFY_FAILED` error.
```
Comment (hexylena) [inline topics/admin/tutorials/tool-management/tutorial.md]:
```suggestion
> If your Galaxy instance is served via the HTTPS protocol (as it should be!), ephemeris will use the [requests](https://requests.readthedocs.io) Python library to encrypt the communication with Galaxy. Therefore, if your Galaxy uses a self-signed SSL certificate, `shed-tools` may fail with a `CERTIFICATE_VERIFY_FAILED` error.
```

**Comment by natefoo** [comment]
:+1: 

Would it be worth mentioning that other than for the purposes of training, this is not something you should ever do on a real system?

**Comment by nsoranzo** [comment]
> Would it be worth mentioning that other than for the purposes of training, this is not something you should ever do on a real system?

@natefoo It does say "if your Galaxy uses a self-signed SSL certificate...", but feel free to amend as you see fit!

**Comment by natefoo** [comment]
True - so it should already be fairly obvious. Although in the case of training, they're not self-signed, they're just signed by an untrusted (and untrustworthy) CA.

**Comment by hexylena** [comment]
Yeah, thought about that same distinction (self vs untrusted) and decided majority won't be bothered and merged as-it-was.

**Pull request merged**